Program.cs

```diff
@@ -8,86 +8,79 @@
 using System.Threading;
 using System.Threading.Tasks;
 using System.Web;
+using static System.String;
 
-namespace TheTroveDownloader
-{
-    static class Program
-    {
+namespace TheTroveDownloader {
+    static class Program {
         private static readonly List<string> IgnoredNames = new List<string>();
         private static readonly List<string> OnlyIncludedNames = new List<string>();
         private static readonly List<string> IgnoredTypes = new List<string>();
-        private static string BasePath = string.Empty;
-        private static int parallel = 5;
-        private static readonly string theTroveUrl = "https://thetrove.net/Books";
+        private static string _basePath = Empty;
+        private static int _parallel = 5;
+        private static string _theTroveUrl = "https://thetrove.net/Books";
 
-        static void Main()
-        {
+        private static void Main() {
             IgnoredNames.Add("Parent Directory");
-            IgnoredNames.Add("?one");            
+            IgnoredNames.Add("?one");
             IgnoredTypes.Add(".DS_Store");
             ReadOptions();
-                        
-            try
-            { 
-                Task.WaitAll(LoadPage(theTroveUrl, BasePath));
+
+            try {
+                Task.WaitAll(LoadPage(_theTroveUrl, _basePath));
                 Console.WriteLine("Finished.");
             }
-            catch(Exception ex)
-            {
+            catch (Exception ex) {
                 Console.WriteLine($"Error: {ex.Message}");
             }
         }
 
-        public static void ReadOptions()
-        {
+        private static void ReadOptions() {
             Console.Write("Enter the path to save files:");
-            BasePath = Console.ReadLine();
+            _basePath = Console.ReadLine()?.Replace('\\', '/').Replace(":\\", "://");
 
-            if(string.IsNullOrWhiteSpace(BasePath))
-            {
-                throw new ArgumentNullException(BasePath, "Invalid path");
+            if (IsNullOrWhiteSpace(_basePath)) {
+                throw new ArgumentNullException(_basePath, "Invalid path");
             }
+            
+            Console.Write("Enter the URL to download from:");
```

Comment:
Wouldn't be best if the asked the url as a optional, leaving books as default?
Or at least inform the possible options to the user.

What do you think?

Reply:
Agreed! I think I intended for that actually but somehow it didn't make it in there... good point!